platform-db-runtime/src/main/java/com/softicar/platform/db/runtime/table/finder/DbTableFinder.java

```diff
@@ -78,8 +79,15 @@ private Field getTableField(Class<?> tableRowClass) {
 		try {
 			return tableRowClass.getField(TABLE_FIELD_NAME);
 		} catch (NoSuchFieldException exception) {
```

Comment:
unrelated change, but without it, lots of unrelated error messages were printed on the console